assets/app/view/game/hex.rb

```diff
@@ -94,7 +94,9 @@ def transform
       def on_hex_click
         return if @actions.empty? && @role != :tile_page
 
-        return store(:tile_selector, nil) if !@clickable || (@hex == @tile_selector&.hex && !@tile_selector.tile)
+        if !@clickable || (@hex == @tile_selector&.hex && !(@tile_selector.respond_to?(:tile) && @tile_selector.tile))
```

Comment:
why do you call respond_to?

Reply:
It's a small bug I found while testing this. The `@tile_selector` variable can actually be a `TokenSelector` and this was throwing an error